app/views/components/_notice.html.erb

```diff
@@ -3,8 +3,15 @@
     description_text ||= ""
     description_govspeak ||= ""
   %>
+
   <section class="app-c-notice" aria-label="Notice">
-    <h2 class="app-c-notice__title"><%= title %></h2>
+
+    <% if description_text.present? || description_govspeak.present? %>
```

Comment:
Not sure if this would work, but I think if you declare description_text ||= false you could then just do if description_text (same with description_govspeak)